crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -729,13 +752,15 @@ mod tests {
     }
 
     #[async_trait::async_trait]
-    impl CheckpointOutput for mpsc::Sender<(CheckpointContents, CheckpointSummary)> {
+    impl CheckpointOutput for mpsc::Sender<(CheckpointContents, CheckpointSummary, bool)> {
         async fn checkpoint_created(
             &self,
             summary: &CheckpointSummary,
             contents: &CheckpointContents,
+            last_checkpoint_of_epoch: bool,
         ) -> SuiResult {
-            self.try_send((contents.clone(), summary.clone())).unwrap();
+            self.try_send((contents.clone(), summary.clone(), last_checkpoint_of_epoch))
+                .unwrap();
```

Comment:
not new code, but this unwrap does not look safe

Reply:
Note that this is test code

Reply:
totally missed that, thanks